qa/workunits/cephtool/test.sh

```diff
@@ -1660,6 +1659,39 @@ function test_mon_osd()
   ceph osd stat | grep up
 }
 
+function test_osd_set_unset()
+{
+  # First, be sure the candidate flags are un-set:
+  ceph osd unset noin noup 
+  ceph osd dump | grep flags | grep -qc -e 'flags.*(noin|noup)' && return 1
+
+  # Set the flags, see that they have been set:
+  ceph osd set noin noup 
+  ceph osd dump | grep -qc -e 'flags.*noin' || return 1
+  ceph osd dump | grep -qc -e 'flags.*noup' || return 1
+
+  # Unset the flags, see that they have been un-set:
+  ceph osd unset noin noup
+  ceph osd dump | grep flags | grep -qc -e 'flags.*(noin|noup)' && return 1
+
+  # Trying to set a non-symmetric flag should cause the whole operation to fail:
+  expect_false ceph osd set noin noup sortbitwise || return 1
+  ceph osd dump | grep flags | grep -qc -e 'flags.*(noin|noup|sortbitwise)' || return 1
```

Comment:
This is failing on my vstart cluster because sortbitwise is set by default on new clusters as far back as infernallis or so.